projects/packages/classic-theme-helper/src/class-main.php

```diff
@@ -39,7 +40,8 @@ public static function init() {
 			self::$instance = new Main();
 			self::$instance->load_modules();
 			// TODO Commenting below since we still load them from theme-tools module
-			// add_action( 'init', array( __CLASS__, 'jetpack_load_theme_tools' ), 30 );
```

Comment:
Might be worth moving the TODO comment further down now so it looks like it is just for the remaining commented out line?